support-frontend/assets/pages/supporter-plus-landing/setup/threeTierConfig.ts

```diff
@@ -0,0 +1,279 @@
+import type { RegularContributionType } from 'helpers/contributions';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+export interface TierBenefits {
+	description?: Array<string | { copy: string; strong: boolean }>;
+	list: Array<{ copy: string; tooltip?: string }>;
+}
+
+export interface PlanCosts {
```

Comment:
`TierPlanCosts` might be a better name as we are considering bundling Plans into a tier